lib/CL/devices/cuda/pocl-cuda.c

```diff
@@ -885,7 +908,7 @@ pocl_cuda_submit_read (CUstream stream, void *host_ptr, const void *device_ptr,
   POCL_MSG_PRINT_CUDA ("cuMemcpyDtoHAsync %p -> %p / %zu B \n", device_ptr, host_ptr, cb);
   CUresult result = cuMemcpyDtoHAsync (
       host_ptr, (CUdeviceptr) (device_ptr + offset), cb, stream);
-  CUDA_CHECK (result, "cuMemcpyDtoHAsync");
+  CUDA_CHECK_ABORT (result, "cuMemcpyDtoHAsync");
```

Comment:
Why abort?

Reply:
`CUDA_CHECK` macro calls abort, and always did that. I'm renaming the macro, to make it clear what it actually does.